app/src/main/res/layout/fragment_validation_success.xml

```diff
@@ -24,12 +24,12 @@
 
```

Comment:
In this file, you can delete or atleast change the raw lottie file. I know it isn’t used, but to avoid confusions later